pkg/updatechecker/updatechecker.go

```diff
@@ -226,7 +228,13 @@ func checkForKotsAppUpdates(opts types.CheckForUpdatesOpts, finishedChan chan<-
 		return nil, errors.Wrap(err, "failed to get app")
 	}
 
-	updateCursor, err := store.GetCurrentUpdateCursor(a.ID, latestLicense.Spec.ChannelID)
+	var licenseChan *kotsv1beta1.Channel
+	licenseChan, err = kotsutil.FindChannelInLicense(a.SelectedChannelID, latestLicense)
```

Comment:
ditto, why not just do this:
```suggestion
	licenseChan, err := kotsutil.FindChannelInLicense(a.SelectedChannelID, latestLicense)
```